internal/db/code.go

```diff
@@ -0,0 +1,114 @@
+package db
+
+import (
+	"fmt"
+	"github.com/google/uuid"
+	"github.com/starlight-go/starlight"
```

Comment:
this import should really not be in the db package, and instead should be a separate package that registers with the executor at server bringup